UserExperience_Remove_CssJs_Page_View.js

```diff
@@ -27,41 +32,41 @@ jQuery(function() {
 				);
 
 				if (!exists) {
-					var single_id = single_path.replace(/[^\w-]/g, '_');
+					var singleID = singlePath.replace(/[^\w-]/g, '_');
 
 					var li = jQuery(
-						'<li id="remove_cssjs_singles_' + single_id + '">' +
+						'<li id="remove_cssjs_singles_' + singleID + '">' +
 						'<table class="form-table">' +
 						'<tr>' +
 						'<th>CSS/JS path to remove:</th>' +
 						'<td>' +
-						'<span class="remove_cssjs_singles_path">' + single_path + '</span>' +
+						'<span class="remove_cssjs_singles_path">' + singlePath + '</span>' +
 						'<input type="button" class="button remove_cssjs_singles_delete" value="Delete"/>' +
 						'</td>' +
 						'</tr>' +
 						'<tr>' +
-						'<th><label for="remove_cssjs_singles_' + single_id + '_action">Behavior:</label></th>' +
+						'<th><label for="remove_cssjs_singles_' + singleID + '_action">Behavior:</label></th>' +
 						'<td>' +
-						'<label class="remove_cssjs_singles_behavior"><input class="remove_cssjs_singles_behavior_radio" type="radio" name="user-experience-remove-cssjs-singles[' + single_path + '][action]" value="exclude" checked>Exclude</label>' +
-						'<label class="remove_cssjs_singles_behavior"><input class="remove_cssjs_singles_behavior_radio" type="radio" name="user-experience-remove-cssjs-singles[' + single_path + '][action]" value="include">Include</label>' +
+						'<label class="remove_cssjs_singles_behavior"><input class="remove_cssjs_singles_behavior_radio" type="radio" name="user-experience-remove-cssjs-singles[' + singlePath + '][action]" value="exclude" checked>Exclude</label>' +
```

Comment:
The text displayed to a user should be translatable.  Use https://developer.wordpress.org/reference/functions/wp_localize_script/ to pass translated text in an object for use in this JavaScript file. 
There are multiple instances of text in this file that should be run through translation functions.

Reply:
Translations added